app/build.gradle

```diff
@@ -90,54 +89,54 @@ android {
 
-    compile('com.crashlytics.sdk.android:crashlytics:2.6.7@aar') {
-        transitive = true;
+    implementation('com.crashlytics.sdk.android:crashlytics:2.7.1@aar') {
+        transitive = true
     }
 
     // Support libraries
-    def supportVersion = "26.1.0";
-    compile "com.android.support:support-v4:$supportVersion"
-    compile "com.android.support:appcompat-v7:$supportVersion"
-    compile "com.android.support:recyclerview-v7:$supportVersion"
-    compile "com.android.support:design:$supportVersion"
+    def supportVersion = "26.1.0"
+    implementation "com.android.support:support-v4:$supportVersion"
+    implementation "com.android.support:appcompat-v7:$supportVersion"
+    implementation "com.android.support:recyclerview-v7:$supportVersion"
+    implementation "com.android.support:design:$supportVersion"
 
     // Google play services
-    def googlePlayVersion = "11.0.4";
-    compile "com.google.android.gms:play-services-base:$googlePlayVersion"
+    def googlePlayVersion = "11.4.2"
+    implementation "com.google.android.gms:play-services-base:$googlePlayVersion"
 
     // Rx
-    compile "io.reactivex.rxjava2:rxjava:2.1.5"
-    compile "io.reactivex.rxjava2:rxandroid:2.0.1"
+    implementation "io.reactivex.rxjava2:rxjava:2.1.5"
+    implementation "io.reactivex.rxjava2:rxandroid:2.0.1"
 
     // Jake Wharton/Square
-    def retrofitVersion = "2.3.0";
-    compile "com.squareup.retrofit2:retrofit:$retrofitVersion"
-    compile "com.squareup.retrofit2:converter-gson:$retrofitVersion"
-    compile "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
-    compile "com.google.code.gson:gson:2.8.2"
+    def retrofitVersion = "2.3.0"
+    implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
+    implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
+    implementation "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
+    implementation "com.google.code.gson:gson:2.8.2"
 
     def okhttpVersion = "3.9.0"
-    compile "com.squareup.okhttp3:okhttp:$okhttpVersion"
-    compile "com.squareup.okhttp3:logging-interceptor:$okhttpVersion"
+    implementation "com.squareup.okhttp3:okhttp:$okhttpVersion"
+    implementation "com.squareup.okhttp3:logging-interceptor:$okhttpVersion"
 
     def butterKnifeVersion = "8.8.1"
-    compile "com.jakewharton:butterknife:$butterKnifeVersion"
+    implementation "com.jakewharton:butterknife:$butterKnifeVersion"
     annotationProcessor "com.jakewharton:butterknife-compiler:$butterKnifeVersion"
-    compile 'com.jakewharton.timber:timber:4.5.1'
-    compile 'com.squareup.picasso:picasso:2.5.2'
+    implementation 'com.jakewharton.timber:timber:4.5.1'
+    implementation 'com.squareup.picasso:picasso:2.5.2'
 
     // Other
-    compile "io.intrepid.commonutils:commonutils:0.2.2"
-    compile 'uk.co.chrisjenx:calligraphy:2.2.0'
+    implementation "io.intrepid.commonutils:commonutils:0.2.2"
+    implementation 'uk.co.chrisjenx:calligraphy:2.2.0'
```

Comment:
We can probably pull Calligraphy out now

Reply:
that will be a separate PR